wiki/Ranking_Criteria/en.md

```diff
@@ -18,11 +18,12 @@ Glossary
 ### Mapset
 
 - **Drain time:** The time during which the health-bar drains. Excludes break periods. Taiko mode’s drain time includes sliders and the intended length of spinners. 
-- **Reasonable Spread:** A mapset without drastically large differences between difficulties as dictated by difficulty-specific rules and guidelines. 
+- **Song Compilation:** An audio file that features multiple different songs or sections of songs.
+- **Play Time**: The time during which the health-bar drains combined with break periods.
+- **Reasonable Spread:** A mapset that does not skip any difficulty levels as dictated by difficulty-specific rules and guidelines, starting from the lowest difficulty level required for the song's length, and does not have any drastically large differences between difficulties. This includes mapsets that feature only a single difficulty.
 - **Game-mode:** osu!, osu!taiko, osu!catch, and each of osu!mania’s key counts are considered individual game-modes.
```

Comment:
`Game-mode` or `Game mode`? Seen multiple instances of each.

`osu!` -> `osu!standard`

Reply:
the actual way to write this is `game mode` with a space